tlspuffin/src/rust_put/openssl/mod.rs

```diff
@@ -218,8 +240,27 @@ impl RustPut {
             ctx_builder.set_groups_list(groups)?;
         }
 
-        // Disallow EXPORT in client
-        ctx_builder.set_cipher_list("ALL:!EXPORT:!LOW:!aNULL:!eNULL:!SSLv2")?;
+        match descriptor.protocol_config.tls_version {
+            TLSVersion::V1_3 => {
+                #[cfg(any(
+                    feature = "openssl101-binding",
+                    feature = "openssl102-binding",
+                    feature = "openssl111-binding",
+                    feature = "libressl-binding"
+                ))]
+                ctx_builder.set_cipher_list(&descriptor.protocol_config.cipher_string_tls13)?;
```

Comment:
please add a comment explaining tha the API changed for those versions, if that is the case

Reply:
Done